### PR TITLE
Feat sort filter follow post

### PR DESCRIPTION
**Pull request by Vishnu-Aithal**
Features - Filter and Sort for Questions and pr-links
Follow Post - Bookmark post Post to follow and recieves notifiatcion

[Live Preview](https://neo-support-git-feat-sort-filter-follow-post-vishnu-aithal.vercel.app/)

### CODE REVIEW

**Review by Developers-codz — APPROVED** [review]
Code and UI both looks good to merge

File: src/routes/PullRequestsPage.jsx [hunk 1]
```diff
@@ -2,24 +2,53 @@ import { Container } from "components/Layout/Container";
...
     usePRLinks(setPullRequests);
 
+    useEffect(() => {
+        if (currentPod === "ALL") {
+            setFilteredPrLinks(prLinks);
+        } else {
+            setFilteredPrLinks(
+                prLinks.filter((link) => link.pod === currentPod)
+            );
+        }
```
Comment (mikesch803) [inline src/routes/PullRequestsPage.jsx:28]: suggestion : here you can use the ternary operator

**Review by mikesch803 — APPROVED** [review]
Nothing find much to suggest, just a suggestion, good to merge

File: src/routes/PullRequestsPage.jsx (hunk 1 above)
Comment (Vishnu-Aithal) [reply src/routes/PullRequestsPage.jsx:28]: Thanks for reviewing, I have kept it for better readability

**Pull request merged**